### DISCUSSION TITLE
Cannot read or change Pentair VS RPM

### DISCUSSION THREAD

**Discussion by RRoyceus** [aqualinkd/AqualinkD#267] [Q&A]
I have  a Pentair VS pump.  Aqualink RS RS8 Combo.
My log shows:
Warning: RS Serial: Serial read bad Pentair checksum, ignoring
Warning: RS Serial:  Read BAD PACKET  Pentair Packet | HEX: 0xff|0x00|0xff|0xa5|0x00|0x21|0x60|0x07|0x0f|0x0a|0x00|0x00|0x05|0x05|0x0b|0x22|0x00|0x00|0x00|0x00|0x00|0x01|0x14|0x22|0x20|0x54|
|

Notice:  RS Serial: RS485 interface received 600 packets in 28 seconds (~21.43 Msg/Sec)
Notice:  RS Serial: Jandy Control Panel Model   :  RS-8 Combo
Notice:  RS Serial: Jandy Control Panel Version :  REV R
Notice:  RS Serial: Jandy ID's found
Notice:  RS Serial: ID 0x50 is in use    <-- Salt Water Generator (Aquarite mode)
Notice:  RS Serial: ID 0x60 is in use    <-- PDA Remote
Notice:  RS Serial: ID 0x08 is not used  <-- can use for Aqualinkd
Notice:  RS Serial: ID 0x09 is not used  <-- can use for Aqualinkd
Notice:  RS Serial: ID 0x0a is not used  <-- can use for Aqualinkd
Notice:  RS Serial: ID 0x0b is not used  <-- can use for Aqualinkd
Notice:  RS Serial: ID 0x40 is not used  <-- can use for Aqualinkd (Extended Device ID)
Notice:  RS Serial: ID 0x41 is not used  <-- can use for Aqualinkd (Extended Device ID)
Notice:  RS Serial: ID 0x42 is not used  <-- can use for Aqualinkd (Extended Device ID)
Notice:  RS Serial: ID 0x43 is not used  <-- can use for Aqualinkd (Extended Device ID)
Notice:  RS Serial: ID 0x30 is not used  <-- can use for Aqualinkd (Prefered Extended Device ID)
Notice:  RS Serial: ID 0x31 is not used  <-- can use for Aqualinkd (Prefered Extended Device ID)
Notice:  RS Serial: ID 0x32 is not used  <-- can use for Aqualinkd (Prefered Extended Device ID)
Notice:  RS Serial: ID 0x33 is not used  <-- can use for Aqualinkd (Prefered Extended Device ID)
Notice:  RS Serial: ID 0x48 is not used  <-- can use for Aqualinkd (RSSA ID)
Notice:  RS Serial: 

Notice:  RS Serial: Pentair ID's found
Notice:  RS Serial: ID 0x21 is in use    <-- Unknown Device
Notice:  RS Serial: ID 0x60 is in use    <-- Pentair VSP
Notice:  RS Serial: 

Notice:  RS Serial: Suggested aqualinkd.conf values
Notice:  RS Serial: -------------------------
Notice:  RS Serial: panel_type =  RS-8 Combo
Notice:  RS Serial: device_id = 0x08
Notice:  RS Serial: rssa_device_id = 0x48
Notice:  RS Serial: extended_device_id = 0x30
Notice:  RS Serial: -------------------------

My button one setings are:
button_01_label=Filter Pump
button_01_pumpID=0x60
button_01_pumpIndex=1

My screen Shows "Filter Pump" "On", but no RPM.  If I long press on it I get a slider, but changing it does not do anything.  I can turn the pump on and off, but when on, it only runs at a high speed, probably, the priming speed.

Can anybody please help?

**Comment by sfeakes** [comment]
First check In aqualinkd.conf do you have those suggested values (and no duplicates).  BTW, the pump info you posted looks correct.

If it's not that, Then you may want to set logging to debug, restart aqualinkd, let it run for a few mins, turn the pump on, wait a few more mins then post the aqualinkd.log file here.
